### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/advanced/additional-responses.md`

### PR DESCRIPTION
**Pull request by javillegasna**
Added Spanish translation to docs/es/docs/advanced/additional-responses.md

### CODE REVIEW

File: docs/es/docs/advanced/additional-responses.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+/// warning
+
+Este es un tema bastante avanzado.
+
+Si apenas estas empezando con **FastAPI**, Puede que no necesites esto.
+
+///
+
+
+Tú puedes declarar respuestas adicionales, que contengan códigos de estado, tipos de medio, descripciones y varias cosas adicionales.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:12]:
```suggestion
Puedes declarar respuestas adicionales, que contengan códigos de estado, tipos de medio, descripciones, etc.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+Ten en cuenta que estas respuestas adicionales se incluirán en el esquema OpenAPI, por lo que también aparecerán en la documentación de la API.
+
+Pero para que esas respuestas adicionales funcionen tienes que asegurarte de que devuelves un `Response` como un objeto`JSONResponse` directamente, el cual debe incluir código de estado y contenido.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:16]:
```suggestion
Pero para que esas respuestas adicionales funcionen, tienes que asegurarte de que devuelves un `Response` como un objeto`JSONResponse` directamente, el cual debe incluir código de estado y contenido.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+## Respuesta Adicional con un `model`
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:18]:
```suggestion
## Respuesta Adicional con `model`
```
o
```suggestion
## Respuesta Adicional usando `model`
```

File: docs/es/docs/advanced/additional-responses.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                        "$ref": "#/components/schemas/HTTPValidationError"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Los schemas están referenciados a otro lugar dentro del esquema de Open Api.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:94]:
```suggestion
Los schemas están referenciados a otro lugar dentro del esquema de OpenAPI.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+        }
+    }
+}
+```
+
+## Tipos de medio adicionales para la respuesta principal
+
+Puedes utilizar los mismos parámetros en tus `responses` para añadir diferentes tipos de medio en la misma respuesta principal.
+
+Por ejemplo, Puedes añadir un tipo de medio adicional de `image/png`, declarando que tu *path operation* pueda retornar un objeto JSON (con el tipo de medio `application/json`) or una imagen PNG:
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:179]:
```suggestion
Por ejemplo, Puedes añadir un tipo de medio adicional de `image/png`, declarando que tu *path operation* pueda devolver un objeto JSON (con el tipo de medio `application/json`) o una imagen PNG:
```

**Review by JavierSanchezCastro — CHANGES_REQUESTED** [review]

File: docs/es/docs/advanced/additional-responses.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
+# Respuestas adicionales en OpenAPI
+
+/// warning
+
+Este es un tema bastante avanzado.
+
+Si apenas estas empezando con **FastAPI**, Puede que no necesites esto.
```
[the remark this answers is not in the document]
Comment (javillegasna) [reply docs/es/docs/advanced/additional-responses.md:7]: Sounds good thanks for the suggestion

File: docs/es/docs/advanced/additional-responses.md (hunk 3 above)
Comment (javillegasna) [reply docs/es/docs/advanced/additional-responses.md:18]: I think I'll go for the second option thx

File: docs/es/docs/advanced/additional-responses.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+
+///
+
+## Combinando información
+
+También puedes combinar la información de la respuesta de multiples lugares, incluyendo los parámetros `response_model`, `status_code`y `responses`.
+
+Puedes declarar un `response_model`, utilizando el código de estado `200` por defecto (o uno personalizado si lo necesitas), y después declarar información adicional para la misma respuesta en `responses`, directamente dentro del schema de OpenAPI.
+
+**FastAPI** mantendrá la información adicional desde `responses`, y la combinará con el esquema JSON desde tu modelo.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:206]:
```suggestion
**FastAPI** mantendrá la información adicional desde `responses` y la combinará con el esquema JSON desde tu modelo.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+
+Por ejemplo, puedes declarar una respuesta con un código de estado `404` que use un modelo de Pydantic y tenga un descripción personalizada.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:208]:
```suggestion
Por ejemplo, puedes declarar una respuesta con un código de estado `404` que use un modelo de Pydantic y tenga una descripción personalizada.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+
+Y una respuesta con un código de estado `200` que use tu `response_model`, pero incluya un ejemplo personalizado dentro de la key `example`:
+
+```Python hl_lines="20-31"
+{!../../../docs_src/additional_responses/tutorial003.py!}
+```
+
+Todo esto sera combinado e incluido dentro de tu OpenAPI, y presentado en la documentación de la API.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:216]:
```suggestion
Todo esto será combinado e incluido dentro de tu OpenAPI, y presentado en la documentación de la API.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+<img src="/img/tutorial/additional-responses/image01.png">
+
+## Combinar respuestas predefinidas y personalizadas
+
+Es posible que desees tener algunas respuestas predefinidas que se apliquen a varios *path operations*, pero quieras combinarlas con respuestas personalizadas que sean necesarias para cada *path operation*.
+
+Para estos casos, puedes usar la técnica de python de "desempaquetar" un  `dict` con `**dict_to_unpack`:
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:224]:
```suggestion
Para estos casos, puedes usar la técnica de Python de "desempaquetar" un `dict` con `**dict_to_unpack`:
```

File: docs/es/docs/advanced/additional-responses.md [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+
+```Python
+{
+    "old key": "old value",
+    "second old key": "second old value",
+    "new key": "new value",
+}
+```
+
+Puedes usar esta técnica para rehusar algunas respuestas predefinidas en tus *path operations* y combinarlas con algunas personalizadas.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:244]:
```suggestion
Puedes usar esta técnica para reutilizar algunas respuestas predefinidas en tus *path operations* y combinarlas con algunas personalizadas.
```

File: docs/es/docs/advanced/additional-responses.md [hunk 12]
```diff
@@ -0,0 +1,257 @@
...
+
+Por ejemplo:
+
+```Python hl_lines="13-17  26"
+{!../../../docs_src/additional_responses/tutorial004.py!}
+```
+
+## Mas información sobre las respuestas de OpenAPI
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:252]:
```suggestion
## Más información sobre las respuestas de OpenAPI
```

File: docs/es/docs/advanced/additional-responses.md [hunk 13]
```diff
@@ -0,0 +1,257 @@
...
+
+Para mirar que exactamente puedes incluir en las respuestas, puedes chequear estas secciones dentro de la especificación de OpenAPI:
```
Comment (JavierSanchezCastro) [inline docs/es/docs/advanced/additional-responses.md:254]:
```suggestion
Para ver exactamente qué puedes incluir en las respuestas, puedes consultar estas secciones dentro de la especificación de OpenAPI:
```

**Review by JavierSanchezCastro — CHANGES_REQUESTED** [review]

File: docs/es/docs/advanced/additional-responses.md [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+
+## Respuesta Adicional usando `model`
+
+Puedes pasar un parámetro `responses` a los *path operation decorators*.
+
+Este parámetro recibe un objeto `dict`, cuyas keys son los códigos de estado para cada respuesta, por ejemplo 200, y  sus valores son otro `dict` con la información necesaria para cada una de las respuestas.
+
+Cada uno de estos `dict`s soporta una key `model`, la cual contenga un modelo de Pydantic, de forma similar al parámetro `response_model`.
+
+**FastAPI** utilizará ese modelo, para generar su propio JSON Schema y lo incluirá en el lugar correcto dentro de OpenAPI.
```
[the remark this answers is not in the document]
Comment (javillegasna) [reply docs/es/docs/advanced/additional-responses.md:26]: mmm I think the first option is better understood

File: docs/es/docs/advanced/additional-responses.md [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+Nota que debes retornar la imagen usando una `FileResponse` directamente.
+
+///
+
+
+/// info
+
+Amenos que especifiques explícitamente en tu parámetro `responses` un tipo de medio diferente, FastAPI asumirá que la respuesta tiene el mismo tipo de medio que las clase de respuesta principal (por defecto `application/json`).
+
+Pero si as especificado una clase de respuesta personalizada with `None` como su tipo de medio, FastAPI usará `application/json` para cualquier respuesta adicional que tenga un modelo asociado.
```
[the remark this answers is not in the document]
Comment (javillegasna) [reply docs/es/docs/advanced/additional-responses.md:196]: sorry I missed that word

**Review by JavierSanchezCastro — APPROVED** [review]

**Comment by javillegasna** [comment]
Hi @alejsdev and @JavierSanchezCastro  It looks like the check-labels job fails, maybe we need to remove **lang-all** label or replace on  .github/workflows/labeler.yml **one_of** check by **any_of**

**Comment by JavierSanchezCastro** [comment]
> Hi @alejsdev and @JavierSanchezCastro It looks like the check-labels job fails, maybe we need to remove **lang-all** label or replace on .github/workflows/labeler.yml **one_of** check by **any_of**

I think it's because @github-actions has added the “docs” tag. I guess if @alejsdev removes that tag, the test will pass correctly.

**Comment by alejsdev** [comment]
It's been fixed! Thanks to both @javillegasna @JavierSanchezCastro

**Comment by javillegasna** [comment]
Maybe there is some update on this?

**Comment by JavierSanchezCastro** [comment]
@javillegasna For my side I approved this PR time ago. You'll need another one to merge.

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**